src/datadog.js

```diff
@@ -0,0 +1,4 @@
+const { DD_ENABLED } = require('./env');
+const tracer = require('dd-trace');
+
+if (DD_ENABLED) tracer.init();
```

Comment:
All of the env vars from the `env` package also need to be passed directly to DD, or reset to the actual process.env, otherwise your defaults will not apply